pkg/minishift/network/utils.go

```diff
@@ -46,10 +51,13 @@ func DetermineHostIP(driver drivers.Driver) (string, error) {
 
 		if NetworkContains(hostaddr, instanceIP) {
 			hostip, _, _ := net.ParseCIDR(hostaddr)
-			if IsIPReachable(driver, hostip.String(), false) {
+			// This step is not working with Windows + VirtualBox as of now
+			// This test is required for CIFS mount-folder case.
```

Comment:
CIFS will not be broken. It is part of a very limited part of the code in that case; host-based Users folder.